sdk/cosmos/azure-cosmos-spark_3-1_2-12/pom.xml

```diff
@@ -29,6 +29,7 @@
     <legal>
       <![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]>
     </legal>
+    <revapi.skip>true</revapi.skip>
```

Comment:
This is a GA library used by customers and shouldn't probably skip revapi checks. 

cc: @kushagraThapar